cheatsheets_draft/Legacy_Application_Management_Cheat_Sheet.md

```diff
@@ -0,0 +1,75 @@
+# Legacy Application Management Cheat Sheet
+
+## Introduction
+
+Legacy applications are applications that are recognised as being outdated but remain in active use by an organisation. This may occur if a viable alternative to the application is not available, replacement is currently too expensive, or if the application is highly bespoke and services a very niche role within an organisation's digital ecosystem. Legacy applications often introduce significant security risks to an organisation for the following reasons:
+
+- Legacy applications might have reached End-of-Life (EoL) meaning that the application no longer receives patching or vendor support. This drastically increases the risk of vulnerability being left in an exploitable state on the application.
```

Comment:
```suggestion
- Legacy applications might have reached End-of-Life (EoL) meaning that the application no longer receives patching or vendor support. This drastically increases the risk of a vulnerability in the application being left in an exploitable state.
```

Reply:
Looks good and a little more readable. Changed. :)